lib/csv/parser.rb

```diff
@@ -1197,20 +1197,12 @@ def parse_row_end
     end
 
     def strip_value(value)
-      return value unless @strip
-      return nil if value.nil?
+      return value if not @strip or value.nil?
```

Comment:
I don't like this unification because of readability.
If you want to show `value` is always returns explicitly, how about just changing to `return value if value.nil?`?

Reply:
Sure, I will modify this to keep both cases separate.